packit/cli/init_local_yaml.py

```diff
@@ -1,65 +1,68 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
 
-import logging
-from pathlib import Path
+# # Copyright Contributors to the Packit project.
+# # SPDX-License-Identifier: MIT
```

Comment:
1. Why are you commenting out even the comments?
2. Why are you even commenting it out?